OvieMudi/Banka
# #165594142 clean up project frontend

#### What does this PR do?
1. Implement feedback by stakeholder
    - add login button to the landing page (header)
    - CTA buttons take users to sign in page
2. Corrects typo in README.md file
3. Fix linting issues on frontend scripts

#### Description of Task to be completed?
- go to the URL at https://oviemudi.github.io/Banka/UI/
- click on the CTA buttons
- ensure the window is taken to the sign page

#### How should this be manually tested?
- clone the repo ` git clone git@github.com:OvieMudi/Banka.git `

- on your terminal run ` cd Banka `

- run ` git checkout ch-165594142-set-up-project-to-consume-api `

- run ` npm install `

- run ` npm run dev:start `

- make a GET request to ` localhost:3000/UI/ `

NB: see screenshot below on the layout to expect

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#165594142](https://www.pivotaltracker.com/story/show/165594142)
#### Screenshots (if appropriate)
![login_link](https://user-images.githubusercontent.com/42499892/56860326-62c69f00-698d-11e9-84f8-3b100a79905b.PNG)
